packages/worker/public/styles.css

```
@@ -955,6 +947,204 @@ html.is-settled {
+	background: var(--color-surface);
+	border: 1px solid var(--color-border);
+	box-shadow:
+		0 10px 24px oklch(from var(--color-text) l c h / 0.1),
+		inset 0 1px 0 oklch(from var(--color-surface) calc(l + 0.06) c h);
+}
+
+.landing-hero-agent-tile::before {
+	content: '';
+	width: 1.7rem;
+	height: 1.7rem;
+	background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)